### PR TITLE
Murano API: Implementing actions API

### PR DESCRIPTION
**Pull request by nmakhotkin**

### CODE REVIEW

**Comment by auhlig** [comment]
Besides the question above this looks very good. Many thanks for contributing @nmakhotkin

**Comment by nmakhotkin** [comment]
Hi @auhlig! 

Sorry but I don't see any question from you.

**Comment by auhlig** [comment]
@nmakhotkin Can you see the comment now?

What do you think on this @vinodborole ?

File: core-test/src/main/java/org/openstack4j/api/murano/v1/ServicesTests.java [hunk 1]
```diff
@@ -74,6 +104,9 @@ public void testCreateOneServiceFromMap() throws IOException {
         ObjectMapper mapper = new ObjectMapper();
         Map<String, Object> serviceMap = mapper.readValue(json, Map.class);
 
+        // Need to remove internal service info from test json string.
+        serviceMap.remove("?");
```
Comment (nmakhotkin) [inline core-test/src/main/java/org/openstack4j/api/murano/v1/ServicesTests.java]: Yes. Let me explain. The whole murano service structure json looks as the following: 

```
{
  "key1": "val1",
  "key2": "val2",
  ...
  "keyN": "valN",
  "?": {
  ..<service internal data like id, type, name etc.>..
  }
}
```

So I divide the application class model onto 2 parts: service internal info (which is all in "?" key) and all other app-related info (arbitrary data). Obviously, serviceMap here contains both data and service info but there is assertion only for data part. Without deletion value in "?" key this assertion won't work.

**Comment by nmakhotkin** [comment]
Thanks, now I see. 

Btw, this PR is WIP for now till I submit action API implementation itself.

**Comment by nmakhotkin** [comment]
@auhlig, @vinodborole, PR is ready for review now.

**Comment by auhlig** [comment]
Nice work.

File: core-test/src/main/java/org/openstack4j/api/murano/v1/ServicesTests.java (hunk 1 above)
Comment (auhlig) [inline core-test/src/main/java/org/openstack4j/api/murano/v1/ServicesTests.java]: Makes sense. Thanks for explaining.

**Comment by nmakhotkin** [comment]
Thanks @auhlig @vinodborole!

**Pull request merged**